### PR TITLE
BZ2100717: Enhances documentation on automatically added node-taints

### PR DESCRIPTION
**Pull request by xenolinux**
Adds a note on automatically added node-taints

Version(s): 4.6+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2100717

Link to docs preview: http://file.rdu.redhat.com/sdudhgao/memory-pressure/nodes/scheduling/nodes-scheduler-taints-tolerations.html#nodes-scheduler-taints-tolerations-about-taintNodesByCondition_nodes-scheduler-taints-tolerations

OR

Preview screenshot https://github.com/openshift/openshift-docs/pull/48266#issuecomment-1198088706

Additional information:

### CODE REVIEW

**Comment by xenolinux** [comment]
Screenshot 2022-07-28 at 6 10 57 PM

File: modules/nodes-scheduler-taints-tolerations-about.adoc [hunk 1]
```diff
@@ -238,6 +238,11 @@ To ensure backward compatibility, the daemon set controller automatically adds t
 
 You can also add arbitrary tolerations to daemon sets.
 
+[NOTE]
+====
+The control plane also adds the `node.kubernetes.io/memory-pressure` toleration on pods that have a QoS class. This is because Kubernetes manages pods in the `Guaranteed` or `Burstable` QoS classes. The new BestEffort pods do not get scheduled onto the affected node.
```
Comment (jeana-redhat) [inline modules/nodes-scheduler-taints-tolerations-about.adoc:243]: Seems "BestEffort" should have some sort of formatting (**bold** or `monospace`?), or be changed to "best effort". If it is a QoS class like `Guaranteed` or `Burstable`, probably `monospace`.

**Review by jeana-redhat — COMMENTED** [review]
One question about "BestEffort" but otherwise LGTM

File: modules/nodes-scheduler-taints-tolerations-about.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/nodes-scheduler-taints-tolerations-about.adoc]: Addressed

**Pull request merged**